plugins/kubernetes/src/components/PodExecTerminal/PodExecTerminal.test.tsx

```diff
@@ -13,20 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import './matchMedia.mock';
-import '@testing-library/jest-dom';
 
 import { DiscoveryApi, discoveryApiRef } from '@backstage/core-plugin-api';
 import {
-  renderWithEffects,
+  renderInTestApp,
   TestApiProvider,
   textContentMatcher,
-  wrapInTestApp,
 } from '@backstage/test-utils';
-import { waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { screen } from '@testing-library/react';
 import WS from 'jest-websocket-mock';
 import React from 'react';
-
+import './matchMedia.mock';
```

Comment:
Noting that ideally this should be in `__testUtils__/matchMediaMock.ts` or something like that

Reply:
I'm following the example in jest docs, let me know if you want the file moved under that subfolder specifically:
https://jestjs.io/docs/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom